transcom/mymove
# B 21326 tget data cleanup int

> [!WARNING]
> This ticket should not be merged to main until https://github.com/transcom/trdm-lambda/pull/28/ has been released to production

### [Ticket B-21326](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1014179)

## Summary

Fixes existing trailing spaces and empty values

### How to test

1. Insert bad dummy data

```sql
INSERT INTO transportation_accounting_codes (
    id, tac, tac_sys_id, loa_sys_id, tac_fy_txt, tac_fnct_poc_nm, created_at, updated_at 
)
VALUES
    ('E08FF3DB-F17D-4076-A9E6-AAB6D3F4029C', 'ABCD', 'dummy', '   dummy', 'dummy ', '  ',now(), now());

INSERT INTO lines_of_accounting (
    id, loa_sys_id, loa_dsc_tx, loa_fnct_prs_nm, loa_agnc_acntng_cd, created_at, updated_at
)
VALUES
    ('E08FF3DB-F17D-4076-A9E6-AAB6D3F4029D', 'dummy', '   dummy', 'dummy ', ' ', now(), now());
```

2. Preview trailing spaces in dummy data
3. Run `make db_dev_migrate`
4. Verify spaces are gone and if `' '` then null
